crates/webidl/src/lib.rs

```diff
@@ -70,6 +72,45 @@ pub fn compile(webidl_source: &str) -> Result<String> {
     Ok(compile_ast(&ast))
 }
 
+/// ChromeOnly is for things that are only exposed to privileged code in Firefox.
+pub fn is_chrome_only(ext_attrs: &Vec<Box<ExtendedAttribute>>) -> bool {
+    ext_attrs.iter().map(|external_attribute| {
+        let mut ret = false;
```

Comment:
You don't need an extra mutable variable here, the `match` block can evaluate to your return value